blameTheISP.py

```diff
@@ -41,32 +47,30 @@ def toggle_ping_only():
 
 def main():
 
-    #speed_test_opts = ConfigJson('~/Blame-The-ISP/blameTheISP.config.json')
-
-    with open(os.path.join(os.path.dirname(os.path.realpath(__file__)), "blameTheISP.config.json"), "r") as config_file: config_data = json.load(config_file)
+    speed_test_opts = ConfigJson('~/Blame-The-ISP/blameTheISP.config.json')
```

Comment:
This assumes the user's directory structure. The config file (by default) should be found as a sibling to this py file, regardless of where that is.

Reply:
Does the sqlite3 database live in that same directory?

Reply:
No, the database should be anywhere, as it is likely to be consumed by other code.